lib/dradis/plugins/nexpose/gem_version.rb

```diff
@@ -8,7 +8,7 @@ def self.gem_version
 
       module VERSION
         MAJOR = 3
-        MINOR = 10
+        MINOR = 11
```

Comment:
Wouldn't this make more sense as v3.10.1? Theoretically, the next big release will be v3.11 so I don't want to get this out of sync with the other v3.10 plugin versions.

Reply:
I would use 3.10.1, but then publish it to rubygems.
Sometimes we have made PR with intermediate versions, but since the version is never really published, when we built the major version, we modify the version to the major one.

Also, we need to update CHANGELOG to 3.10.1 and fix the month to September/October